modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -10,21 +10,26 @@ class IntentToFileController < BaseFormController
         include ClaimsApi::PoaVerification
 
         before_action { permit_scopes %w[claim.write] }
-        before_action :validate_json_format, if: -> { request.post? }
         before_action :validate_json_schema, only: %i[submit_form_0966 validate]
-        before_action :check_future_type, only: [:submit_form_0966]
 
         FORM_NUMBER = '0966'
         def submit_form_0966
-          response = itf_service.create_intent_to_file(form_type)
-          render json: response['intent_to_file'],
+          response = bgs_service.intent_to_file.insert_intent_to_file(intent_to_file_options)
+          render json: response,
                  serializer: ClaimsApi::IntentToFileSerializer
         end
 
         def active
-          response = itf_service.get_active(active_param)
-          render json: response['intent_to_file'],
-                 serializer: ClaimsApi::IntentToFileSerializer
+          bgs_response = bgs_service.intent_to_file.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
+            target_veteran.participant_id,
+            ClaimsApi::IntentToFile::ITF_TYPES[active_param]
+          )
+          if bgs_response.present?
+            render json: bgs_response&.first,
```

Comment:
if `bgs_response` is present, i don't think we need the safe navigation (`&.`)

Reply:
Done